internal/scaffold/olm-catalog/descriptor/parse_test.go

```diff
@@ -58,11 +65,16 @@ func TestParseResource(t *testing.T) {
 			"Resource string with unquoted name",
 			`"Memcached,v1,memcached.example.com"`, v1alpha1.APIResourceReference{}, true,
 		},
+		{
+			"Resource string literal with unquoted name",
+			"`Memcached,v1,memcached.example.com`", v1alpha1.APIResourceReference{}, true,
+		},
 	}
 
 	for _, c := range cases {
 		output, err := parseResource(c.input)
 		if err != nil {
+			fmt.Printf("%s: %v\n", c.description, err)
```

Comment:
Was this `fmt.Printf()` line meant to be added or was this for debug?

Reply:
Debug, removing